Easy/1. Shopping Cart Problem/solutions/README.md

```diff
@@ -1 +1,2 @@
 # Solutions
+![alt text](https://github.com/jsuryakt/code-with-girlscript-bangalore/blob/ShoppingCartEasy/Easy/1.%20Shopping%20Cart%20Problem/solutions/jsuryakt_Jayasurya/Screenshot%20(24).png?raw=true)
```

Comment:
These two lines are not required. Please don't modify any files outside your folder.

Reply:
Ok sorry 
I'll remove that.